package.json

```diff
@@ -3,18 +3,16 @@
   "name": "root",
   "repository": "git@github.com:connectrpc/connect-query-es.git",
   "scripts": {
-    "build": "pnpm --recursive build",
-    "generate": "pnpm --recursive generate",
-    "test": "pnpm --recursive test",
-    "lint": "pnpm run lint:code && pnpm run lint:markdown && pnpm run lint:spelling",
-    "lint:fix": "pnpm run lint:code:fix && pnpm run lint:markdown:fix && pnpm run lint:spelling",
-    "lint:markdown": "markdownlint-cli2 \"**/*.md\" \"#**/node_modules\"",
-    "lint:markdown:fix": "markdownlint-cli2-fix \"**/*.md\" \"#**/node_modules\"",
-    "lint:spelling": "cspell \"**\" --gitignore",
-    "lint:code": "eslint . && prettier . --check && license-header --check",
-    "lint:code:fix": "eslint . --fix && prettier . --write && license-header",
-    "all": "pnpm run build && pnpm run generate && pnpm run lint:fix && pnpm run test",
-    "update-all": "pnpm outdated --recursive || pnpm update --recursive --latest && pnpm outdated --recursive"
+    "build": "turbo run build",
+    "generate": "turbo run generate",
+    "test": "turbo run test",
+    "format": "turbo run format:all",
+    "format:all": "pnpm run format:code && pnpm run format:markdown && pnpm run check:spelling",
+    "format:markdown": "markdownlint-cli2-fix \"**/*.md\" \"#**/node_modules\"",
+    "check:spelling": "cspell \"**\" --gitignore",
+    "format:code": "eslint . --fix && prettier . --write && license-header",
```

Comment:
If you use eslint ignores, prettier sometimes shifts them. So I would recommend to always format first, then run eslint.

eslint also requires dependencies in the monorepo to be built. My best idea for that would be to make use of turbo dependencies, but that requires you to lint per package. turbo will parallelize when possible though, so it might end up being faster in many situations.

Reply:
I think my only concern for moving all these formatting/linting commands into the packages is how it handles files at the root level (README.md, jest-preset.js, etc)... I guess i could configure the root to run a separate series of commands to format those...